twoplayer-tetris-PRG.s.diff

```diff
@@ -1,6 +1,22 @@
 --- orig
 +++ mod
-@@ -33,10 +33,18 @@ score           := $0053
+@@ -3,10 +3,15 @@ ; Created:    2020-01-12 13:38:44
```

Comment:
So whats the best way to solve it for know? The change of the diff does not make it very clear, what I wanted to change. I basically pulled in my layout adresses and replaced the constants sent to PPU before doing line count rendering. As I added lines a the very beginning its cluttered with all those line numbers changing.

Reply:
I think for now we just accept your changes. #3 will fix the date. The line number shifting has always been unfortunate.

I think there are some things we can do to avoid inserting lines. (1) have a single common include at the top. We can include more things transitively as appropriate (2) We use variables more. For example, things get quite a bit cleaner if I would set INGAME_LAYOUT_P1_LEVEL_X for the other two layouts as well, with ifdefs only in the include files instead of in the disassembly. Unfortunately, that won't be enough; at times we will still need ifdefs to choose the modifications to make and I don't see a good solution to that other than only one of us make such changes at a time.